addon/model.js

```diff
@@ -514,6 +514,9 @@ export default class MegamorphicModel extends EmberObject {
   // Errors hash that will get update,
   // upon validation errors
   get errors() {
+    if (this._errors === null) {
```

Comment:
I think the reason for this was that errors is sometimes set/manipulated by addons so we shouldn't be messing with that if thats the case. Might be safer to still make _errors lazy and stash in __errors

Reply:
Since there is no setter for `errors` it was not previously possible to set it except by setting `_errors` which would still work here.